bin/run-css-lint

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+const child_process = require( 'child_process' );
+
+var args = [ '**/*.scss'," --syntax scss"];
```

Comment:
Prefer `let` over `var`.  Also, shouldn't `--syntax` and `scss` be separate arguments?

```js
let args = [ '**/*.scss', '--syntax', 'scss' ];
```